### PR TITLE
各種ページの修正

### PR DESCRIPTION
**Pull request by Omuretsu**
・表題のとおりです。
・各種ページにおける細かい修正を行いました。
・修正内容については、各ページのコミット名とファイルにて確認をおねがいします。

### CODE REVIEW

**Comment by jaotan** [comment]
# 変更のあったファイルプレビュー
- [blog/skin-change.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/blog/skin-change)
- [server/guidelines/griefing.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/guidelines/griefing)
- [server/policies/bans.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/policies/bans)
- [server/rules/discord.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/rules/discord)
- [server/specifications.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/specifications)
- [support/faq.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/support/faq)

File: server/guidelines/griefing.md [hunk 1]
```diff
@@ -38,4 +38,4 @@ Regular権限以上の方は、必要に応じてChatBanを実行して構いま
 
 ## 運営側の対応
 
-[各種処罰方針](/server/policies/bans)をご覧ください。
+利用者の迷惑行為を確認した場合、運営は[各種処罰方針](/server/policies/bans)に基づき対処をします。
```
Comment (book000) [inline server/guidelines/griefing.md:41]:
```suggestion
利用者の迷惑行為を確認した場合、運営は[各種処罰方針](/server/policies/bans)に基づき対応いたします。
```
「運営側の対応」なので、「対処」ではなく「対応」の方が読みやすいかと。

File: server/specifications.md [hunk 2]
```diff
@@ -76,9 +76,16 @@ updatedAt: 2021-09-07
 ## 投票について
 
 当サーバは[minecraft.jp](https://minecraft.jp/servers/play.jaoafa.com)と[monocraft.net](https://monocraft.net/servers/4ovU0v9PkdyJbNJVngf7)に登録しており、各サイトからの投票を受け付けています。  
-投票方法や投票特典については、[こちら](/blog/how-to-vote)のブログ記事をご覧ください。
+投票方法や投票特典については、[こちら](/blog/how-to-vote)のブログ記事をご覧ください。  
+なお上記2つのサイトについては、jao Minecraft Server とは別団体が運営を行っています。
```
Comment (book000) [inline server/specifications.md:80]:
```suggestion
なお上記2つのサイトは、jao Minecraft Server が運営するWebサイトではありませんのでご注意ください。
```

これがいいかどうかもわからんけど、「jao Minecraft Server とは別団体が運営を行っています。」は違和感があるかと…(せめて `～とは別の団体が～`)。

File: server/specifications.md [hunk 3]
```diff
@@ -76,9 +76,16 @@ updatedAt: 2021-09-07
 ## 投票について
 
 当サーバは[minecraft.jp](https://minecraft.jp/servers/play.jaoafa.com)と[monocraft.net](https://monocraft.net/servers/4ovU0v9PkdyJbNJVngf7)に登録しており、各サイトからの投票を受け付けています。  
-投票方法や投票特典については、[こちら](/blog/how-to-vote)のブログ記事をご覧ください。
+投票方法や投票特典については、[こちら](/blog/how-to-vote)のブログ記事をご覧ください。  
+なお上記2つのサイトについては、jao Minecraft Server とは別団体が運営を行っています。  
+
+稀に、投票特典が正常に付与されないことがあります。投票から3週間(24日)以内であれば、運営への補填申請の元、補填します。[公式Discordサーバ](/blog/join-discord)`#support`へ、以下を送信ください。
```
Comment (book000) [inline server/specifications.md:82]:
```suggestion
稀に、投票特典が正常に付与されないことがあります。投票から3週間(24日)以内であれば、運営への補填申請を行っていただければ補填対応いたします。[公式Discordサーバ](/blog/join-discord)の`#support`へ、以下を送信ください。  
```

補填に関する話ってこの一文しか今までなかったっけ…？

**Review by book000 — CHANGES_REQUESTED** [review]

File: server/policies/bans.md [hunk 4]
```diff
@@ -3,10 +3,11 @@ title: 各種処罰方針
 description: 各種禁止処置(処罰・Ban)についての方針やその種類を記載します。
 image: https://storage.jaoafa.com/a2a9f8dcd472210cb808bc9432bc6ae3.png
 createdAt: 2018-01-03
-updatedAt: 2021-09-07
+updatedAt: 2021-09-21
 ---
 
-このページでは、jao Minecraft Server（以下、「当サーバ」といいます）おける各種禁止処置(処罰・Ban)について記載します。
+このページでは、jao Minecraft Server（以下、「当サーバ」といいます）おける各種禁止処置(処罰・Ban)について記載します。
```
Comment (book000) [inline server/policies/bans.md:9]:
```suggestion
このページでは、jao Minecraft Server（以下、「当サーバ」といいます）における各種禁止処置(処罰・Ban)について記載します。  
```

**Review by book000 — CHANGES_REQUESTED** [review]

**Comment by jaotan** [comment]
# 変更のあったファイルプレビュー
- [blog/skin-change.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/blog/skin-change)
- [server/guidelines/griefing.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/guidelines/griefing)
- [server/policies/bans.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/policies/bans)
- [server/rules/discord.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/rules/discord)
- [server/specifications.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/specifications)
- [support/faq.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/support/faq)

**Comment by jaotan** [comment]
# 変更のあったファイルプレビュー
- [blog/skin-change.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/blog/skin-change)
- [server/guidelines/griefing.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/guidelines/griefing)
- [server/policies/bans.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/policies/bans)
- [server/rules/discord.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/rules/discord)
- [server/specifications.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/specifications)
- [support/faq.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/support/faq)

File: server/specifications.md (hunk 3 above)
Comment (Omuretsu) [reply server/specifications.md:82]: 調べた限り、こことDiscordの #info とかぐらい？

Comment (book000) [reply server/specifications.md:82]: まあここに今回の説明追加すればいいか…

**Comment by jaotan** [comment]
# 変更のあったファイルプレビュー
- [blog/skin-change.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/blog/skin-change)
- [server/guidelines/griefing.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/guidelines/griefing)
- [server/policies/bans.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/policies/bans)
- [server/rules/discord.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/rules/discord)
- [server/specifications.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/specifications)
- [support/faq.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/support/faq)

File: server/specifications.md (hunk 3 above)
Comment (Omuretsu) [reply server/specifications.md:82]: そうしたら上記のsuggestionをcommitすればよい？

**Comment by jaotan** [comment]
# 変更のあったファイルプレビュー
- [blog/skin-change.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/blog/skin-change)
- [server/guidelines/griefing.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/guidelines/griefing)
- [server/policies/bans.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/policies/bans)
- [server/rules/discord.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/rules/discord)
- [server/specifications.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/specifications)
- [support/faq.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/support/faq)

File: server/specifications.md (hunk 3 above)
Comment (book000) [reply server/specifications.md:82]: y

**Comment by jaotan** [comment]
# 変更のあったファイルプレビュー
- [blog/skin-change.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/blog/skin-change)
- [server/guidelines/griefing.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/guidelines/griefing)
- [server/policies/bans.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/policies/bans)
- [server/rules/discord.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/rules/discord)
- [server/specifications.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/specifications)
- [support/faq.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/support/faq)

**Review by book000 — APPROVED** [review]

**Comment by jaotan** [comment]
# 変更のあったファイルプレビュー
- [blog/skin-change.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/blog/skin-change)
- [server/guidelines/griefing.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/guidelines/griefing)
- [server/policies/bans.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/policies/bans)
- [server/rules/discord.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/rules/discord)
- [server/specifications.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/server/specifications)
- [support/faq.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/82/support/faq)

**Pull request merged**